src/components/pages/AccountSettings/RenderAccountSettings.js

```diff
@@ -1,29 +1,28 @@
 import React, { useState, useEffect } from 'react';
 import { Modal, Button, Form } from 'antd';
-import { useOktaAuth } from '@okta/okta-react';
+import { useAuth0 } from '@auth0/auth0-react'; //replaces okta
```

Comment:
Same comment as above about the commented out message.

Reply:
Great catch @SethMacP!